app/src/main/res/layout/custom_theme_dialog.xml

```diff
@@ -29,6 +29,7 @@
                     android:layout_height="wrap_content"
                     android:layout_weight="1"
                     android:textSize="25dp"
+                    android:textColor="@color/white"
```

Comment:
does this work when the device theme is light mode?

Reply:
hmm as much as i know on every theme background of alert dialog is black, and yes, my device on white theme